DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/javascriptIndexes/javascriptMapIndex/javascriptMapIndex.java

```diff
@@ -0,0 +1,56 @@
+
+
+import net.ravendb.client.documents.indexes.AbstractJavaScriptIndexCreationTask;
+import net.ravendb.client.documents.session.IDocumentSession;
+
+import net.ravendb.demo.common.DocumentStoreHolder;
+import java.util.List;
+import com.google.common.collect.Sets;
+
+public class javascriptMapIndex {
```

Comment:
Class names in Java upper case -> the same applies to file name, see https://github.com/ravendb/demo/pull/114

Reply:
Done

Reply:
Done